scripts/LedSignMock.py

```diff
@@ -0,0 +1,36 @@
+
+import curses
+import argparse
+
+parser = argparse.ArgumentParser(description="Prints mock output.")
+parser.add_argument('--mock-output', help = "Displays Mock Output", )
+args = parser.parse_args()
+
+menu = ['   ','SCE LED Sign Visualization ', ' ','Most Recent Health Check: ', 'Most Recent Sign Update: ',' ',
+'Current Sign Text:', 'Current Brightness:', 'Current Scroll Speed:', 'Current Text Color: ', 'Current Background:',
+'Current Border Color: ' ]
+
+class LedSignMock(data):
+    def print_menu(stdscr, selected_row_idx):
+        data = "info here"
+        stdscr.clear()
+        stdscr.border(0)
+        h, w = stdscr.getmaxyx()
+        for idx, row in enumerate(menu):
+            x = w//3 - len(menu)
+            y = h//3 - len(menu)//3 + idx
+            stdscr.addstr(y, x, row)
+        stdscr.refresh()
+
+    def main(stdscr):
+        curses.curs_set(0)
+        current_row = 0
+        print_menu(stdscr, current_row)
+        x = stdscr.getch()
+        while x != 27:
```

Comment:
what is `27` mean? it would be better to define `27` somewhere in the class like:

```py
class LedSignMock:
  WHATEVER_27_MEANS = 27

  # later on...
  if x == self.WHATEVER_27_MEANS:
    # handle the case here, much more readable
```

Reply:
27 represents the esc button in curses, not something i have defined. Do you want me to put that in as a comment?